buffer/src/main/java/io/netty5/buffer/api/pool/PoolChunk.java

```diff
@@ -178,6 +178,7 @@ final class PoolChunk implements PoolChunkMetric {
     private final int chunkSize;
 
     int freeBytes;
+    int pinnedBytes;
```

Comment:
why not using a var handle setOpaque/getOpaque (or acquire/release too) here? modifying it is supposed to be single writer (locked) hence we can save a synchronized get making metrics gathering a bit more lightweight